massa-hash/src/hash.rs

```diff
@@ -24,6 +51,10 @@ impl PartialOrd for Hash {
     }
 }
 
+/// This function is useful for the BTreeMap where the order of the addresses is to be maintained.
```

Comment:
Just a tiny specification that it is not only used for addresses

Reply:
Fixed